.github/workflows/test-dandi-cli.yml

```diff
@@ -14,6 +14,7 @@ jobs:
     env:
       NO_ET: 1
       DANDI_ALLOW_LOCALHOST_URLS: 1
+      DJANGO_DANDI_WEB_APP_URL: https://dandiarchive.org/
```

Comment:
hm, should it go into tests fixture or wouldn't otherwise we need to set this env var in every environment where we would run tests?

Reply:
dandischema reads the envvar when the dandischema package is loaded, which is before pytest fixtures take effect, so we need to set the envvar outside the Python process.

Reply:
ah, right. thanks! Let's proceed then, and see where else we would need to do that for now.